### PR TITLE
[FIX] "Invalid interface clause" error in UFW module (#2559)

### PR DESCRIPTION
**Pull request by ovcharenko**
##### ISSUE TYPE
- Bugfix Pull Request
##### COMPONENT NAME

`system/ufw.py`
##### ANSIBLE VERSION

```
ansible 2.2.0
```
##### SUMMARY

Fixed `interface` and `log` order in `ufw` call.

Before the order was wrong:

```
/usr/sbin/ufw allow *log in* from any to any port 22 proto tcp
```

Producing the error noticed by reporter:

```
ERROR: Invalid interface clause
```

After:

```
/usr/sbin/ufw allow *in log* from any to any port 22 proto tcp
```

And it adds rule as expected.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @ovcharenko. This module is going into community review.

We notice that you are one of the maintainers (@ovcharenko, @pyykkis) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by ovcharenko** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @ovcharenko. Since you are one of the maintainers (@ovcharenko, @pyykkis) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: system/ufw.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 short_description: Manage firewall with UFW
 description:
     - Manage firewall with UFW.
-version_added: 1.6
+version_added: 1.7
```
Comment (resmo) [inline system/ufw.py]: this should not be modified usually. what was the reason you changed it?

Comment (ovcharenko) [inline system/ufw.py]: @resmo I updated version just to reflect all recent changes baseline.

Comment (resmo) [inline system/ufw.py]: `version_added` is not used for versioning the module, it is an information in which version of ansible the module was added. could you please change it back?

**Pull request merged**